core/src/data_sources/qdrant.rs

```diff
@@ -10,16 +12,37 @@ use serde::{Deserialize, Serialize};
 pub enum QdrantCluster {
     #[serde(rename = "main-0")]
     Main0,
-    //#[serde(rename = "dedicated-0")]
-    //Dedicated0,
+    #[serde(rename = "dedicated-0")]
+    Dedicated0,
 }
 
-static QDRANT_CLUSTER_VARIANTS: &[QdrantCluster] = &[QdrantCluster::Main0];
+static QDRANT_CLUSTER_VARIANTS: &[QdrantCluster] =
+    &[QdrantCluster::Main0, QdrantCluster::Dedicated0];
+
+impl ToString for QdrantCluster {
```

Comment:
out of scope but I am suprised that we have to implement these ourselves everytime, there must be a serde (or other) thing to do this for us